packages/ember-data/lib/system/store.js

```diff
@@ -185,6 +185,7 @@ Store = Ember.Object.extend({
       store: this
     });
     this._pendingSave = [];
+    this._containerCache = Ember.create(null);
```

Comment:
on destroy, we should be sure to also destroy these. So they can do needed cleanup